lib/project-helpers.ts

```diff
@@ -1,6 +1,50 @@
-export interface CreateProjectValues {
-  name: string;
-  goal: string;
-  owner: string;
-  champion: string;
+import { useEffect } from "react";
+import { useState } from "react";
+import { Project } from "../db/models/project";
+
+export class ProjectValues {
```

Comment:
This can be an interface since you only need the type

Reply:
I've just had a look at the usage. Potentially, yes, it could just be an interface since the class doesn't provide any functionality via methods etc.